go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/florianl/go-conntrack v0.4.0
 	github.com/golang/protobuf v1.5.4
 	github.com/google/go-cmp v0.7.0
+	github.com/google/nftables v0.2.0
```

Comment:
hum, question here: I can see there's a 0.3.0 tag, with a bunch of changes. Is it worth using it instead?